src/main/java/nextstep/ladder/view/ResultView.java

```diff
@@ -0,0 +1,39 @@
+package nextstep.ladder.view;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import nextstep.ladder.domain.Lines;
+import nextstep.ladder.view.dto.PrintPlayerNamesDto;
+
+public class ResultView {
+
+    private static final String LINE_FLAG = "-----";
+    private static final String LINE_BLANK = "     ";
+    private static final String LINE_DELIMITER = "|";
+
+    private static final String LINE_PREFIX = LINE_BLANK + LINE_DELIMITER;
+
+    public static void printPlayerNames(PrintPlayerNamesDto dto) {
+        dto.getPlayerNames().forEach(name -> System.out.printf("%6s", name));
+        System.out.println();
+    }
+
+    public static void printLadders(Lines lines) {
+        IntStream.range(0, lines.getHeight())
+            .mapToObj(i -> createLineBody(lines.getExists(i)))
+            .map(s -> LINE_PREFIX + s)
+            .forEach(System.out::println);
+    }
+
+    private static String createLineBody(List<Boolean> lineExists) {
+        return lineExists.stream()
+            .map(e -> {
+                if (e) {
+                    return LINE_FLAG;
+                }
+
+                return LINE_BLANK;
+            }).collect(Collectors.joining(LINE_DELIMITER)) + LINE_DELIMITER;
```

Comment:
```suggestion
            .map(e -> e ? LINE_FLAG : LINE_BLANK)
            .collect(Collectors.joining(LINE_DELIMITER)) +
            LINE_DELIMITER;
```
이럴땐 삼항연산자를 사용해도 괜찮을 것 같습니다.
혹은 아예 메소드로 분리하는 방법도 있어요!

Reply:
네 저도 실무라면 삼항 연산자를 썻을 것 같은데, 이전 리뷰에서 삼항 연산자도 else 라고 하셔서 사용을 안했네요 ㅠ